structs.go

```diff
@@ -582,37 +582,79 @@ type VoiceState struct {
 
 // A Presence stores the online, offline, or idle and game status of Guild members.
 type Presence struct {
-	User   *User    `json:"user"`
-	Status Status   `json:"status"`
-	Game   *Game    `json:"game"`
-	Nick   string   `json:"nick"`
-	Roles  []string `json:"roles"`
-	Since  *int     `json:"since"`
+	// The user the presence updated for.
+	User *User `json:"user"`
+
+	// Roles the user is in.
+	Roles []string `json:"roles"`
+
+	// The status of the user.
+	Status Status `json:"status"`
+
+	// Either nil or the user's current activity.
+	Game *Activity `json:"game"`
+
+	// The user's platform-dependent status.
+	ClientStatus *ClientStatus `json:"client_status"`
+
+	// User's current activities.
+	Activities []*Activity `json:"activities"`
 }
 
-// GameType is the type of "game" (see GameType* consts) in the Game struct
-type GameType int
+// ActivityType is the type of "activity" (see ActivityType* consts) in the Activity struct
+type ActivityType int
 
 // Valid GameType values
 const (
-	GameTypeGame GameType = iota
-	GameTypeStreaming
-	GameTypeListening
-	GameTypeWatching
+	ActivityTypeGame ActivityType = iota
```

Comment:
This is a breaking change, which we do not allow

Reply:
I will change the names back to Game in that case.